src/core/lib/iomgr/pollset_custom.h

```diff
@@ -19,13 +19,14 @@
 #ifndef GRPC_CORE_LIB_IOMGR_POLLSET_CUSTOM_H
 #define GRPC_CORE_LIB_IOMGR_POLLSET_CUSTOM_H
 
+#include "src/core/lib/iomgr/error.h"
 #include <grpc/support/port_platform.h>
 
 #include <stddef.h>
 
 typedef struct grpc_custom_poller_vtable {
   void (*init)();
-  void (*poll)(size_t timeout_ms);
+  grpc_error* (*poll)(size_t timeout_ms);
```

Comment:
@lidizheng As a change to the custom iomgr interface, do you know if this will impact Objective C?

Reply:
ObjC's cfstream integration is several level higher at the endpoint interface. This PR won't impact ObjC, AFAIK.